cpp/tests/grouped_rolling/grouped_rolling_test.cpp

```diff
@@ -1214,7 +1214,7 @@ TYPED_TEST(GroupedTimeRangeRollingTest,
 TYPED_TEST(GroupedTimeRangeRollingTest, SimplePartitionedStaticWindowsWithNoGroupingKeys)
 {
   const size_type DATA_SIZE{static_cast<size_type>(6)};
-  const std::vector<TypeParam> col_data(DATA_SIZE, 1);
+  const std::vector<TypeParam> col_data(DATA_SIZE, TypeParam(1));
```

Comment:
wouldn't you have to include the duration types for this typed test (`GroupedTimeRangeRollingTest`) for these changes to be necessary?